components/style/properties/helpers.mako.rs

```diff
@@ -447,32 +447,45 @@
 
         impl<'a> ToCss for LonghandsToSerialize<'a> {
             fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
-                let mut all_flags = SerializeFlags::all();
+                let mut _all_flags = SerializeFlags::all();
```

Comment:
Nit: Maybe have `#[allow(unused_variables)]` at the top of the module, instead of underscoring everywhere?